src/RealPath.php

```diff
@@ -42,48 +29,18 @@ public function filter(mixed $value): mixed
         if (! is_string($value)) {
             return $value;
         }
-        $path = (string) $value;
-
-        if ($this->pathMustExist) {
-            return realpath($path);
-        }
 
-        ErrorHandler::start();
-        $realpath = realpath($path);
-        ErrorHandler::stop();
-        if ($realpath !== false) {
-            return $realpath;
+        if ($this->pathMustExist && ! file_exists($value)) {
+            return $value;
         }
 
-        $drive = '';
-        if (stripos(PHP_OS, 'WIN') === 0) {
-            $path = preg_replace('/[\\\\\/]/', DIRECTORY_SEPARATOR, $path);
-            if (preg_match('/([a-zA-Z]\:)(.*)/', $path, $matches)) {
-                [, $drive, $path] = $matches;
-            } else {
-                $cwd   = getcwd();
-                $drive = substr($cwd, 0, 2);
-                if (! str_starts_with($path, DIRECTORY_SEPARATOR)) {
-                    $path = substr($cwd, 3) . DIRECTORY_SEPARATOR . $path;
-                }
-            }
-        } elseif (! str_starts_with($path, DIRECTORY_SEPARATOR)) {
-            $path = getcwd() . DIRECTORY_SEPARATOR . $path;
```

Comment:
Conditionally preprending `getcwd()` is a feature we should probably keep and test:

```php
$filter->filter('foo.txt'); // "/var/www/project/foo.txt"
```